perception/traffic_light_occlusion_predictor/README.md

```diff
@@ -8,22 +8,24 @@ For each traffic light roi, hundreds of pixels would be selected and projected i
 
 ![image](images/occlusion.png)
 
-If no point cloud is received or all point clouds have very large stamp difference with the camera image, the occlusion ratio of each roi would be set as 0.
+If no point cloud is received or all point clouds have very large stamp difference with the camera image, the occlusion ratio of each roi would be set as 0. The signal whose occlusion ratio is larger than max_occlusion_ratio will be set as unknown type.
 
 ## Input topics
 
-| Name                 | Type                                                | Description              |
-| -------------------- | --------------------------------------------------- | ------------------------ |
-| `~input/vector_map`  | autoware_auto_mapping_msgs::HADMapBin               | vector map               |
-| `~/input/rois`       | autoware_auto_perception_msgs::TrafficLightRoiArray | traffic light detections |
-| `~input/camera_info` | sensor_msgs::CameraInfo                             | target camera parameter  |
-| `~/input/cloud`      | sensor_msgs::PointCloud2                            | LiDAR point cloud        |
+| Name                                 | Type                                                | Description                      |
+| ------------------------------------ | --------------------------------------------------- | -------------------------------- |
+| `~input/vector_map`                  | autoware_auto_mapping_msgs::HADMapBin               | vector map                       |
+| `~/input/car/traffic_signals`        | tier4_perception_msgs::msg::TrafficLightArray       | vehicular traffic light signals  |
+| `~/input/pedestrian/traffic_signals` | tier4_perception_msgs::msg::TrafficLightArray       | pedestrian traffic light signals |
```

Comment:
We are discussing to unify in the direction of Traffic Light instead of Traffic Signal, so could you please change it to light? cc @mitsudome-r

Reply:
Do the names of topics need to change?

Reply:
@yukkysaito Friendly ping

Reply:
@tzhong518 
I agree with what @yukkysaito  said, that since we've changed all the traffic signal types to traffic light types, we should also change the topic name accordingly.
In this PR, or split into separate PRs, would it be possible for you to address the issue?